enrichment_wrangler.py

```diff
@@ -77,16 +78,21 @@ def lambda_handler(event, context):
 
         # Create parameter json from environment variables.
         parameters = {"marine_mismatch_check": marine_mismatch_check,
+                      "survey_column": survey_column,
                       "period_column": period_column,
                       "identifier_column": identifier_column}
 
         logger.info("Retrieved data from s3")
         data_json = data_df.to_json(orient="records")
+        json_payload = {
+            "data": json.dumps(data_json),
+            "lookups": lookup_info,
+            "parameters": json.dumps(parameters),
+            "survey_column": survey_column
+        }

```

Comment:
This has been done slightly differently to other modules as it has the 'parameters' inside the payload alongside other variables, whereas others just have them all in the payload. This should probably be made to match perhaps?

Reply:
Assuming there was a reason for the parameters and was pre-existing so rather than change completely just added the survey_colum to these. May want to change, however. @kingmushroom ?

Reply:
I think change would be best as I can't see a good reason for it as far as I can tell. I assume it is just leftover from earlier and was never changed to match.

Reply:
I'd never seen it prior so just assumed it was added for a reason. Can someone please confirm as I have no idea. @kingmushroom @Jordancooke

Reply:
David is right. Parameters are runtimevariables recieved and passed along. I think the decision was made at the time to keep them seperate to the lookupinfo that enrichment recieved. 

`        # Create parameter json from environment variables.
        parameters = {"marine_mismatch_check": marine_mismatch_check,
                      "period_column": period_column,
                      "identifier_column": identifier_column}
`
No reason they need to be in their own blob. I think it was tidier at the time, but now doesnt match up with the way other things are working.

Reply:
Rightio. Will get to work.